### ISSUE TITLE
Add 'geocoded' event

### ISSUE THREAD

**Issue by AW-UC** [perliedman/leaflet-routing-machine#143]
If used with a Geocoder, the input fields will update the location name once the coordinates of a dragged marker are reverse geocoded. It would be great to have an event firing, once the `name` is obtained.

**Comment by AW-UC** [comment]
FYI: at the moment I am achieving the desired result by adding this as a final line to `_updateGeocoder`:
```
this.fire('waypointGeocoded', {waypoint: wp, value: value, element: geocoderElem});
```
Now, `L.Routing.Plan` has a `waypointGeocoded` event. I could submit a PR, but I guess you prefer a different approach.

**Comment by perliedman** [comment]
This sounds like good thing, I'll think some about it.

My only comment would be that Leaflet event names use all lower case as standard, so the event name should be `waypointgeocoded`.

**Comment by perliedman** [comment]
This is now implemented by 9d862041b10360af3d879555d4f3334c8b9c56b5.

**Comment by AW-UC** [comment]
Great! Thanks a lot! Needed a small fix, though. See #157
